dnadoh/assemble.py

```diff
@@ -48,9 +49,8 @@ def _read_phenotypes(stem):
     filename = util.filename_phenotypes(stem)
     return pd.read_csv(filename)
 
-
-if __name__ == "__main__":
-    import sys
-
-    combined = read_combined(sys.argv[1])
-    print(combined)
+def assemble(root, write_csv=False):
+    combined = read_combined(root)
+    if write_csv:
+        combined.to_csv(f"{sys.argv[1]}-assembled.csv", index=False)
```

Comment:
i think it's a bit brave to use `sys.argv[1]` here without checking :-)

Reply:
Updated to use the function argument (this file/these functions are never meant to be called from the CLI)